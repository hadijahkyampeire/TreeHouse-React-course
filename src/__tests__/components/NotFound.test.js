import React from 'react';
import enzyme from 'enzyme';
import NotFound from '../../components/NotFound';

describe('About component', () => {
    const node = enzyme.shallow(<NotFound/>);

    it('should render without crashing', () => {
        expect(node).toMatchSnapshot();
    });
});