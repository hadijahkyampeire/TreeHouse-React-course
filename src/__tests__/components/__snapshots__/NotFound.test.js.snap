// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component should render without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <i
          className="material-icons icn-error"
>
          error_outline
</i>,
        <h2>
          Page Not Found
</h2>,
      ],
      "className": "main-content not-found",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "error_outline",
          "className": "material-icons icn-error",
        },
        "ref": null,
        "rendered": "error_outline",
        "type": "i",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Page Not Found",
        },
        "ref": null,
        "rendered": "Page Not Found",
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <i
            className="material-icons icn-error"
>
            error_outline
</i>,
          <h2>
            Page Not Found
</h2>,
        ],
        "className": "main-content not-found",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "error_outline",
            "className": "material-icons icn-error",
          },
          "ref": null,
          "rendered": "error_outline",
          "type": "i",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Page Not Found",
          },
          "ref": null,
          "rendered": "Page Not Found",
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
